init-tools.sh

```diff
@@ -103,6 +106,18 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
         cd $__DOTNET_PATH
         tar -xf $__DOTNET_PATH/dotnet.tar
 
+        # Install .NET Core 1.1 until we get CLI updated to a version that has it bundled in
+        echo "Installing .NETCore 1.1..."
+        __NETCORE11_LOCATION="https://dotnetcli.blob.core.windows.net/dotnet/release/1.1.0/Binaries/1.1.0/${__NETCORE11_PKG}.1.1.0.tar.gz"
+
+        which curl > /dev/null 2> /dev/null
```

Comment:
Echoing what was done here: https://github.com/dotnet/coreclr/pull/8823, please use POSIX `command -v` instead of non-standard `which` command. It requires one-less dependency to install on CentOS7 and ArchLinux etc.